ceph-releases/ALL/debian/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -0,0 +1,9 @@
+# add the necessary repos
+    echo "deb http://obs.linaro.org/ERP:/ceph-ansible:/${CEPH_VERSION}/Debian___ENV_[BASEOS_TAG]__ ./" \
+      >> /etc/apt/sources.list.d/erp.list && \
+    echo "deb http://deb.debian.org/debian/ __OS_CODENAME__-backports main" \
```

Comment:
Sounds like an extra-space before __OS_CODENAME__

Reply:
That space is expected. The string should be this "deb http://deb.debian.org/debian/ stretch-backports main"